### ISSUE TITLE
Referenced documents in toctree have similar name - confusing

### ISSUE THREAD

**Issue by yoonghm** [googleapis/python-texttospeech#20]
https://googleapis.dev/python/texttospeech/latest/index.html has the following entries

![image](https://user-images.githubusercontent.com/4215991/77167159-6eb0d780-6af0-11ea-885e-9cff69e7329c.png)

It is not obvious the version of the documents. I would like to propose the following changes in https://github.com/googleapis/python-texttospeech/edit/master/docs/index.rst

```
.. include:: README.rst

Api Reference
-------------
.. toctree::
    :maxdepth: 2

    Text-to-Speech Client API Reference <api>
    Client for Cloud Text-to-Speech API (v1) <gapic/v1/api>
    Types for Cloud Text-to-Speech API Client (v1) <gapic/v1/types>
    Client for Cloud Text-to-Speech API (v1beta1) <gapic/v1beta1/api>
    Types for Cloud Text-to-Speech API Client (v1beta1) <gapic/v1beta1/types>
```

**Comment by yoonghm** [comment]
See pull request in https://github.com/googleapis/python-texttospeech/pull/21

**Comment by munkhuushmgl** [comment]
@yoonghm I have approved and merged your PR #21

**Comment by munkhuushmgl** [comment]
@yoonghm FYI the documentation will be updated on the next release.
